src/components/Icon/Icon.tsx

```diff
@@ -201,6 +201,7 @@ export const BUNDLED_ICONS = {
   delete: deleteIcon,
   export: exportIcon,
   import: importIcon,
+  minusIcon: minusMinor,
```

Comment:
@DhruvinPatel78 Does icon exist with name minusIcon? I think minusMinor icon exists and an export alias name can be created as minusMinor: minusIcon.